descriptor.md

```diff
@@ -28,13 +28,14 @@ The following describe the primary set of properties that make up a _Descriptor_
   This property exists so that a client will have an expected size for the content before validating.
   If the length of the retrieved content does not match the specified length, the content SHOULD NOT be trusted.
 
-### Reserved
+- **`urls`** *array*
 
-The following are field keys that MUST NOT be used in descriptors specified in other OCI specifications:
+  This OPTIONAL property specifies a list of URLs from which this object can be downloaded.
```

Comment:
On Thu, Aug 25, 2016 at 07:48:50AM -0700, Stephen Day wrote:
> >  
> > -The following are field keys that MUST NOT be used in descriptors specified in other OCI specifications:
> > +  This OPTIONAL property specifies a list of URLs from which this object can be downloaded.
> 
> In this case, we are referring to a specific object, since we have type and size.

The caller may know the type and size, but it's only downloading the
blob.  And that only meets your “free-form binary data” criterion [1].

[1]: https://github.com/opencontainers/image-spec/pull/94#discussion_r64955841